src/server/service/index.ts

```diff
@@ -0,0 +1,28 @@
+import { prisma } from "@server/db/client";
+import { google } from "googleapis";
+import { env } from "@env/server.mjs";
+
+export const createDriveService = async (userID: string) => {
```

Comment:
Good modularity! I think this should throw if `findFirst` returls `null`, since any subsequent action on `drive` would fail.